products/db-knownprojects/sca_aggregate/boundaries_es_zone.sql

```diff
@@ -12,332 +12,459 @@ drop table if exists aggregated_es_zone_project_level;
 drop table if exists longform_es_zone_output;
```

Comment:
This file was made all lowercase sql keywords, while others are uppercase. @alexrichey has theory that maybe it does it per file, depending on capitalization of first keyword? Would be nice to test, and maybe see if we can specify that in the template/settings just to make sure it's always consistent.

Reply:
@fvankrieken @damonmcc https://docs.sqlfluff.com/en/stable/rules.html#rule-capitalisation.keywords 

Looks like we can just set the policy to upper case.